template/composer.json

```diff
@@ -27,7 +27,7 @@
     "behat/mink-goutte-driver":     "*",
     "behat/mink-selenium2-driver":  "*",
     "behat/mink-browserkit-driver": "*",
-    "drush/drush":                  "dev-master",
+    "drush/drush":                  "^9.0",
```

Comment:
Should we stick with `^8.0` for now? I think 9.0 is basically the current 'experimental' branch undergoing a full rearchitecture.